attributes/default.rb

```diff
@@ -75,8 +75,6 @@
       '2.4'
     end
   when 'freebsd'
-    node['platform_version'].to_f >= 10.0 ? '2.4' : '2.2'
-  else
```

Comment:
i mean you are removing the `else`, just making sure thats what you want since this will have no default anymore